.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+install:
+  - pip install -r requirements.txt
+  - git clone https://github.com/simphony/JYU-LB.git
```

Comment:
Oops, my mistake, I thought that it was a python package similar to simphony. No it is ok like this. What I would prefer that this set of commands that download and built jyulib could be moved into a bash script (like https://github.com/simphony/simphony-openfoam/pull/4/files#diff-a82724aa77d86db6f25e85b2ca756d7a)